lib/container.js

```diff
@@ -181,6 +181,16 @@ export default class Container {
     });
   }
 
+  whoami() {
+    let container = this;
+    return container.makeRequest('me', {})
+    .then(function (res) {
+      return new container.User(res.result);
+    }, function (err) {
+      return Promise.reject(err);
+    });
```

Comment:
I think `whoami` will fetch the remote user AND update the `_user` (i.e. keep the `_currentUser` in sync with remote.